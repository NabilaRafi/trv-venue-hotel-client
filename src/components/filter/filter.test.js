import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { Provider } from 'react-redux';
import configureStore from '../../store/store';
import { shallow , configure} from 'enzyme';
import FilterComponent from './index';

configure({adapter: new Adapter()});

describe('Render Card component', () => {
    const wrapper = shallow(<Provider store={configureStore}><FilterComponent /></Provider>);
    it(' should render card component without crashing', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
