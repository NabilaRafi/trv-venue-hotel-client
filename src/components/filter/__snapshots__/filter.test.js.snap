// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Card component  should render card component without crashing 1`] = `ShallowWrapper {}`;
